bonfire/Modules/Settings/Controllers/GeneralSettingsController.php

```diff
@@ -76,6 +76,30 @@ public function saveGeneral()
         setting('App.dateFormat', $this->request->getPost('dateFormat'));
         setting('App.timeFormat', $this->request->getPost('timeFormat'));
 
+        // Check for an logo to upload
+        if ($file = $this->request->getFile('siteLogo')) {
+
+            if ($file->isValid()) {
+
+                $filename = 'custom.'. $file->getExtension();
+
+                if ($file->move(ROOTPATH .'public/uploads/logo', $filename, true)) {
```

Comment:
Please save in `WRITEPATH`.'uploads'. The `asset` helper can be used to load it later.  I don't want to force people to make the public folder writable.